core/src/test/java/org/apache/iceberg/TestManifestWriterVersions.java

```diff
@@ -152,9 +160,10 @@ public void testV2WriteWithInheritance() throws IOException {
     checkEntry(readManifest(manifest), SEQUENCE_NUMBER, SEQUENCE_NUMBER, FileContent.DATA);
   }
 
-  @Test
-  public void testV2WriteDelete() throws IOException {
-    ManifestFile manifest = writeDeleteManifest(2);
+  @ParameterizedTest
+  @ValueSource(ints = {2, MAX_FORMAT_VERSION})
```

Comment:
I would probably use ALL_VERSIONS here too and just do an assume not 1

Reply:
Updated.